### ISSUE TITLE
Seekur Jr Model

### ISSUE THREAD

**Issue by jhoare** [MobileRobots/amr-ros-config#1]
I'm going to be using a Seekur Jr for an upcoming project, and I was wondering it would be possible for you to release a `.dae` file that is textured for visualization in rviz and in gazebo. 

I have found a `.stl` file in [this repository](https://github.com/kuri-kustar/seekur_jr_robot), but I'm hoping to get a textured model. 

I've noticed in this repository, the Seekur Jr is just a Box, and the dimensions do not seem to match the actual dimensions of the robot.

**Comment by reed-adept** [comment]
Yes, this model needs a lot of work.  We have a Solid Works CAD model at http://robots.mobilerobots.com/CADModels/SeekurJr/ but not sure what it has for texture info.  You could load the mesh into a tool like MeshLab, Blender, etc. and just color the faces if that might be good enough for RViz.   Let me know if you need any info about the robot useful for the model, or any changes in the original Solid Works CAD model and I can see if I can find someone who can help with that.

**Comment by DavidHan008** [comment]
how can I use it?
